net/mosquitto/patches/010-openssl-deprecated.patch

```diff
@@ -0,0 +1,46 @@
+--- a/lib/net_mosq.c
++++ b/lib/net_mosq.c
+@@ -87,9 +87,11 @@ int net__init(void)
+ #endif
+ 
+ #ifdef WITH_TLS
++#if OPENSSL_VERSION_NUMBER < 0x10100000L
```

Comment:
You can have lower versions with different deprecations, are you sure you shouldbn't be using the openssl compat defines? not the version defines?

Reply:
Fairly. wget currently breaks because it uses it.